pkg/services/live/features/dashboard.go

```diff
@@ -0,0 +1,80 @@
+package features
+
+import (
+	"encoding/json"
+
+	"github.com/centrifugal/centrifuge"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+// DashboardEvent events related to dashboards
+type dashboardEvent struct {
+	UID       string `json:"uid"`
+	Action    string `json:"action"` // saved, editing
+	UserID    int64  `json:"userId,omitempty"`
```

Comment:
Working with event systems before, it's always nice if the events are information-rich and more usable standalone.

It would mean including more dashboard & user info here like dashboard name, (maybe a user object, that includes id, login, email, avatarUrl) makes consuming and using these events much easier without requiring further API calls.

Reply:
We could add the whole json model... But I think we would still need to do a full refresh in the current system regardless.  If only dashboard were an observable 🙂

Reply:
I was more thinking of other uses for this event (like an admin viewing a list of activity), and for the modal, we might want to show the user name & profile image of the user who updated the same dashboard you have. 

So ash much meta info about the entities in the event is usually good. Could also be used by a search or cache subsystem for example to update index state and for that we do not want to issue further db/api calls.

Reply:
I'm happy to add more info now -- but since this is an internal event, I think we should wait and add more info when we have a real use for it.

Reply:
> I was more thinking of other uses for this event 

That is info that should be attached on the 'presense' side -- that still needs to be explored some more, but I fear this PR is getting too big as is :)